.github/workflows/pr_tests.yml

```diff
@@ -63,41 +59,68 @@ jobs:
         # Run tests from integration_tests sub dir
         working-directory: ./integration_tests
     strategy:
+      fail-fast: false
       matrix:
         dbt_version: ["1.*"]
-        warehouse: ["bigquery", "snowflake", "databricks", "redshift"] # TODO: Add RS self-hosted runner
-
-    # services:
-    #   postgres:
-    #     image: postgres:latest
-    #     env:
-    #       POSTGRES_DB: ${{ secrets.POSTGRES_TEST_DBNAME }}
-    #       POSTGRES_USER: ${{ secrets.POSTGRES_TEST_USER }}
-    #       POSTGRES_PASSWORD: ${{ secrets.POSTGRES_TEST_PASS }}
-    #     # Set health checks to wait until postgres has started
-    #     options: >-
-    #       --health-cmd pg_isready
-    #       --health-interval 10s
-    #       --health-timeout 5s
-    #       --health-retries 5
-    #     ports:
-    #       # Maps tcp port 5432 on service container to the host
-    #       - 5432:5432
+        warehouse: ["bigquery", "snowflake", "databricks", "redshift", "spark_iceberg"] # TODO: Add RS self-hosted runner
+    services:
+      postgres:
```

Comment:
Is this actually used anywhere? I thought those were commented out in the end because postgres is not supported. I think we can remove them altogether.

Reply:
No